scss/partials/_people.scss

```diff
@@ -0,0 +1,29 @@
+.bureau-people {
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  text-align: center;
+}
+
+.people-photo {
+  float: left;
+  margin: 0 20px 30px 20px;
+}
+
+.people-photo img {
+  max-height: 200px;
+  margin-bottom: 10px;
```

Comment:
Vertical spaces doesn`t work for inline elements (like <img>, <span>, https://www.w3resource.com/html/HTML-block-level-and-inline-elements.php List of inline elements)